geoips/commandline/ancillary_info/cmd_instructions.yaml

```diff
@@ -73,24 +131,24 @@ instructions:
       - Package Path
       - Source Code
       - Version Number
-  geoips_get_plugin:
-    help_str: |
-      Retrieve the appropriate GeoIPS Plugin alongside descriptive information of
-      that plugin. See output_info for each datum provided when this command is
-      called. For a listing of available GeoIPS Plugins, run:
-      `geoips list plugins`.
-    usage_str: |
-      To use, type `geoips get plugin <interface_name> <plugin_name>`, where
-      <interface_name> is a valid GeoIPS Interface and <plugin_name> is a valid plugin
-      identifier that has been implemented in any installed GeoIPS package.
-    output_info:
-      - Docstring
-      - Documentation Link (If applicable)
-      - Family
-      - Interface Name
-      - Package
-      - Relative Path
-      - Plugin Specific Info (signature / source_names / available overrides)
+  # geoips_get_plugin:
```

Comment:
I'm leaning towards keeping it until we've made a final decision on what the help messages will contain, a list of stable commands, etc. Better to have it and not use it rather than making code changes and having to rewrite it. That being said, I'm happy to remove it if that's what we want to do.